<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNew</class>
 <widget class="QWidget" name="AddNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_addLine">
    <attribute name="title">
     <string>添加线路</string>
    </attribute>
    <widget class="QLabel" name="label_inputLineName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入线路名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_inputLineName">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_selectColor">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择线路颜色：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_selectColor">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_inputSSname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入起始站名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_inputSSname">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_SSlongitude">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>起始站经度(东经)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SSlatitude">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>起始站纬度(北纬)：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_SSlongitude">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_SSlatitude">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_inputESname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入终点站名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_inputESname">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ESlongitude">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>终点站经度(东经)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ESlatitude">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>460</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>终点站纬度(北纬)：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_ESlongitude">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>410</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_ESlatitude">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>460</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_addLine">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>271</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addLine">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>460</y>
       <width>150</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_addStation">
    <attribute name="title">
     <string>添加站点</string>
    </attribute>
    <widget class="QLabel" name="label_inputSname">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>站点名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Sname">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_inputSlongitude">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>站点经度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_inputSlatitude">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>站点纬度：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Slongitude">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>120.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Slatitude">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_LinesIn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>301</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_selectLineIn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加所属线路：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_addStation">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>321</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addStation">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>500</y>
       <width>150</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_addEdge">
    <attribute name="title">
     <string>添加边</string>
    </attribute>
    <widget class="QLabel" name="label_S1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>站点1：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_S2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>站点2：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_linesIn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>所属线路：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_linesIn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_S2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_S1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_addEdge">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>351</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addEdge">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>410</y>
       <width>150</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
